styles/sass/base/_typography.scss

```diff
@@ -0,0 +1,31 @@
+h1 {
+    font-size: 80px;
+    font-weight: 500;
+    line-height: 1.25em;
+    margin: 0px;
+    color: #1a1a1a;
+}
+
+h2 {
+    font-size: 56px;
+    font-weight: 500;
+    line-height: 1.25em;
+    padding-bottom: 56px;
+    color: #1a1a1a;
+}
+
+.large-body {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 1.5;
+    padding-bottom: 40px;
+    color: #4a4a4a;
+}
+
+.small-body {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 1.5;
+    margin-left: 16px;
+    color: #4a4a4a;
+}
```

Comment:
this file doesn't pass the prettier check. Probably missing an empty line on the end. Please run `npx prettier styles/sass/base/_typograpthy.scss` and commit changes